src/validation.cpp

```diff
@@ -4010,19 +4010,22 @@ bool CVerifyDB::VerifyDB(
         CBlock block;
         // check level 0: read from disk
         if (!ReadBlockFromDisk(block, pindex, consensus_params)) {
-            return error("VerifyDB(): *** ReadBlockFromDisk failed at %d, hash=%s", pindex->nHeight, pindex->GetBlockHash().ToString());
+            LogPrintf("VerifyDB(): *** ReadBlockFromDisk failed at %d, hash=%s\n", pindex->nHeight, pindex->GetBlockHash().ToString());
+            return VerifyDBResult::ERROR_CORRUPTED_BLOCK_DB;
         }
         // check level 1: verify block validity
         if (nCheckLevel >= 1 && !CheckBlock(block, state, consensus_params)) {
-            return error("%s: *** found bad block at %d, hash=%s (%s)\n", __func__,
-                         pindex->nHeight, pindex->GetBlockHash().ToString(), state.ToString());
+            LogPrintf("%s: *** found bad block at %d, hash=%s (%s)\n", __func__,
```

Comment:
nit in e7dfc743406bfa0cac94c640dd5eaeac4364c990: (here and elsewhere)

When changing this from `error`, which included `ERROR: ` in the log message, it could make sense to keep some kind of mention of "error" and instead remove the function name, which the user can already inject via `-logsourcelocations`.